### PR TITLE
chore: Keep our privates private once again.

### PR DESCRIPTION
**Pull request by seanhealy**
### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- Private components are once again hidden away safely.

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: doczrc.js [hunk 1]
```diff
@@ -50,15 +50,34 @@ const projectPlugin = () =>
     },
   });
 
+/**
+ * Atlantis contains some private components. These are indicated by an
+ * empty `.private` file in the component's directory.
+ */
 function privateComponentReadmies() {
```
Comment (JeffMarvin) [inline doczrc.js]: Speeeling... Either this
```suggestion
function privateComponentReadmes() {
```
or something like
```suggestion
function privateComponentVisibility() {
```

File: doczrc.js [hunk 2]
```diff
@@ -50,15 +50,34 @@ const projectPlugin = () =>
     },
   });
 
+/**
+ * Atlantis contains some private components. These are indicated by an
```
Comment (JeffMarvin) [inline doczrc.js]: What does this function do? That should be the first part of the comment. The other notes are nice (and can be included) but don't explain the why of the function.

**Comment by seanhealy** [comment]
@JeffMarvin As per npm convention, you can always [find commands in the `package.json`](https://github.com/GetJobber/atlantis/blob/master/package.json#L8) I don't know that it should be in the readme?

**Comment by JeffMarvin** [comment]
> @JeffMarvin As per npm convention, you can always [find commands in the `package.json`](https://github.com/GetJobber/atlantis/blob/master/package.json#L8) I don't know that it should be in the readme?

Agreed, the package.json is the right place for it.

LGTM

**Pull request merged**